engine/schema/src/main/resources/META-INF/db/schema-41700to41710.sql

```diff
@@ -17,4 +17,7 @@
 
 --;
 -- Schema upgrade from 4.17.0.0 to 4.17.1.0
---;
\ No newline at end of file
+--;
+
+UPDATE `cloud`.`configuration` set `value` = 'false' where `name` = 'network.disable.rpfilter' and `value` != 'true';
```

Comment:
```suggestion
UPDATE `cloud`.`configuration` SET `value` = 'false' WHERE `name` = 'network.disable.rpfilter' AND `value` != 'true';
```

Same for the other update...